activesupport/lib/active_support/callbacks.rb

```diff
@@ -663,10 +663,12 @@ def __update_callbacks(name) #:nodoc:
       #
       # ===== Options
       #
-      # * <tt>:if</tt> - A symbol naming an instance method or a proc; the
-      #   callback will be called only when it returns a +true+ value.
-      # * <tt>:unless</tt> - A symbol naming an instance method or a proc; the
-      #   callback will be called only when it returns a +false+ value.
+      # * <tt>:if</tt> - A symbol, a string or an array of symbols and strings,
+      #   each naming an instance method or a proc; the callback will be called
+      #   only when they all return a +true+ value.
```

Comment:
A think we should not quote `true` as we are referring to Ruby's definition of a "true value" and not the exact value `true`.